fairroot/base/steer/FairRunAna.cxx

```diff
@@ -531,7 +532,7 @@ void FairRunAna::RunTSBuffers()
             fRootManager->ReadNonTimeBasedEventFromBranches(globalEvent++);
         }
         fTask->ExecuteTask("");
-        FillEventHeader();
+        GetSource()->FillEventHeader(GetEventHeader());
```

Comment:
I think, this one is causing the final segmentation fault, because GetSource might return a `nullptr`. The old `FillEventHeader()` catches this (withut giving a warning…)